world/stat_checks/check_maker.py

```diff
@@ -20,9 +20,15 @@ def check_rolls_tied(roll1, roll2, tie_value=TIE_THRESHOLD):
 
 class SimpleRoll:
     def __init__(
-        self, character=None, stat=None, skill=None, rating: DifficultyRating = None
+        self,
+        character=None,
+        stat=None,
+        skill=None,
+        rating: DifficultyRating = None,
+        receivers: list = None,
```

Comment:
typehint of list with a default of None probably raises linting errors because you're not saying it can be empty. The way to do that is with the `Union` value from the typing library, which also has its own container types so that you can specify the values inside them. So it'd be something like:

`receivers: Union[List, None] = None`

With List, you could also specify `List[str]` to show that it'd be a list of names. More on that later though...